linkerd/linkerd2
# Upgrading packages and updating package.json to reflect current versions

Recent `package.json` upgrades (#2818) triggered warnings about `react-router` and `react-router-dom`'s deprecated use of `require("history/FILENAME")`. This PR upgrades `react-router` and `react-router-dom` so that the warnings are not triggered.

To test: 
1. From `master`, run `bin/web setup` then `bin/web test`. (You can also run `bin/web dev` and look at your dashboard console.) The tests will pass but you will see multiple warnings like the below:
```
Please use require("history").createBrowserHistory instead of require("history/createBrowserHistory"). Support for the latter will be removed in the next major release.
```
2. Check out this branch, run `bin/web setup` then `bin/web test`. The tests will pass with no console warnings.
